kratos/includes/master_slave_constraint.h

```diff
@@ -223,7 +223,7 @@ class MasterSlaveConstraint
      * @param Constant The constant in the master slave relation
      * @return A Pointer to the new constraint
      */
-    virtual MasterSlaveConstraint::Pointer Create(
+    KRATOS_DEPRECATED_MESSAGE("This is legacy version, please remove it") virtual MasterSlaveConstraint::Pointer Create(
```

Comment:
Already removing this function should not cause any problems right ?

Reply:
assuming this is overridden in derived classes those will no longer compile
Thats Why I would first deprecate it, then remove the usages, and finally remove it from the BaseClass

Reply:
Ah I think you meant sth else
Yes it can be safely removed, it is never called any more, since Variables of this type were removed completely in #6686

Reply:
So shall we remove it directly then ?

Reply:
As I wrote then we'd have to also remove it from the derived classes, which I would feel more comfortable if we did it in separate PRs
Or imagine some closed apps like the one of Altair use this, then we directly break their build

Reply:
Ok .. fine by me !